statistics/histogram.go

```diff
@@ -392,6 +392,23 @@ func (hg *Histogram) getIncreaseFactor(totalCount int64) float64 {
 	return float64(totalCount) / float64(columnCount)
 }
 
+// validRange checks if the range is valid, it is used by `SplitRange` to remove the invalid range.
+func validRange(ran *ranger.NewRange) bool {
+	var low, high []byte
+	if ran.LowVal[0].Kind() == types.KindBytes {
+		low, high = ran.LowVal[0].GetBytes(), ran.HighVal[0].GetBytes()
+	} else {
+		low, high = codec.EncodeInt(nil, ran.LowVal[0].GetInt64()), codec.EncodeInt(nil, ran.HighVal[0].GetInt64())
```

Comment:
How about uint?

Reply:
When [encoding](https://github.com/pingcap/tidb/blob/master/distsql/request_builder.go#L176) the handle key, we also only consider int.

Reply:
Please comment it.

Reply:
Done.

Reply:
The range for request will only contain the [MinInt64, MaxInt64] part or the [MaxInt64+1, MaxUint64] part. It's already split beyond there.